Source/WebCore/platform/mediastream/ios/AVAudioSessionCaptureDeviceManager.mm

```diff
@@ -220,7 +220,7 @@ - (void)routeDidChange:(NSNotification *)notification
         if (currentInput != m_lastDefaultMicrophone.get()) {
             auto device = AVAudioSessionCaptureDevice::create(currentInput, currentInput);
             callOnWebThreadOrDispatchAsyncOnMainThread(makeBlockPtr([device = crossThreadCopy(WTFMove(device))] () mutable {
-                CoreAudioSharedUnit::singleton().handleNewCurrentMicrophoneDevice(WTFMove(device));
+                CoreAudioSharedUnit::unitSingleton().handleNewCurrentMicrophoneDevice(WTFMove(device));
```

Comment:
This is a small point, but I'd prefer "CoreAudioSharedUnit::singleton" as the standardized name. "Unit" is already in the name, so it doesn't add information. Stand-alone "singleton" helps to clarify that this function returns the only CoreAudioSharedUnit. Otherwise, if there's a CoreAudioSharedUnit::unitSingleton, maybe there's also a CoreAudioSharedUnit::somethingElseSingleton?

Reply:
Good point--fixed!